doc/openocd.texi

```diff
@@ -11631,6 +11631,12 @@ riscv exec_progbuf 0x0330000f 0x0000100f
 riscv exec_progbuf 0x94a20405
 @end example
 
+@deffn {Command} {riscv autofence} [on|off]
+When on (default), some situations, such as stepi, will automatically execute fence.
+When off, user needs to take care of the cache themselves, for example, using the
+riscv exec_progbuf to execute fence or cmo command.
+@end deffn
+
```

Comment:
Please, could you write the documentation in more accurate manner:

- What instructions are actually executed.
- What exactly is "some situations".
- There is nothing in OpenOCD called `stepi`.
- `cmo` should be spelled as `CMO` and explained to the reader what it stands for
- `the cache` --> `memory coherency`

For example:

```suggestion
@deffn {Command} {riscv autofence} [on|off]
When on (default), OpenOCD will automatically execute RISC-V fence instructions
(@var{fence.i} and @var{fence rw, rw}) in these cases:

@itemize @bullet

@item before step or resume,
@item before memory read via the Program Buffer,
@item after memory write via the Program Buffer.

@end itemize

When off, users needs to take care of memory coherency themselves, for example
using the @var{riscv exec_progbuf} command to execute fences or CMO instructions
(RISC-V Cache Management Operations).
@end deffn

```

Reply:
Thanks,  this doc is much better than before